src/client/components/ResultMessageModalWindow/ResultMessageModalWindow.styles.css

```diff
@@ -0,0 +1,70 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body{
+    background-image: url(../../assets/images/background/background-treasure-map.png);
```

Comment:
Background image for your component is google map image, not the treasure map image. And it is okay to use body for background image for testing but at the time of submitting code we should remove this type of global css, because it will effect all other components.

Reply:
Done